src/lib/config/index.js

```diff
@@ -1,3 +1,6 @@
 export { getSearchConfig } from './searchConfig';
 export { invenioConfig } from './invenioConfig';
 export { uiConfig } from './uiConfig';
+
+// NOTE: Support for external configuration, merge with defaults
```

Comment:
not sure I understand the comment, in any case you will have to refactor it when providing overridable config...